packages/webapp/src/pages/Account/InviteSignup.tsx

```diff
@@ -1,182 +1,119 @@
-                name: user.name,
-                accountId: user.accountId
-            });
-            signin(user);
+        const declined = await apiDeclineInvite(token!);
+        if (declined && declined.res.status === 200) {
+            toast({ title: `You declined the invitation`, variant: 'success' });
             navigate('/');
-        } else if (res != null) {
-            const errorMessage = (await res.json()).error.message || 'Unknown error';
-            setServerErrorMessage(errorMessage);
+        } else {
+            toast({ title: 'An unexpected error occurred', variant: 'error' });
         }
+        setLoadingDecline(false);
     };
 
-    return (
-        <>
+    if (loading) {
+        return (
             <DefaultLayout>
                 <div className="flex flex-col justify-center">
-                    <div className="flex flex-col justify-center w-80 mx-4">
-                        <h2 className="mt-2 text-center text-3xl font-semibold tracking-tight text-white">Sign up</h2>
-                        <form className="mt-6 space-y-6" onSubmit={handleSubmit}>
-                            <div>
-                                <div className="mt-1">
-                                    <input
-                                        id="name"
-                                        name="name"
-                                        type="text"
-                                        autoComplete="name"
-                                        required
-                                        placeholder="Name"
-                                        minLength={1}
-                                        maxLength={100}
-                                        className="border-border-gray bg-dark-600 placeholder-dark-500 text-text-light-gray block h-11 w-full appearance-none rounded-md border px-3 py-2 text-[14px] placeholder-gray-400 shadow-sm focus:outline-none"
-                                    />
-                                </div>
-                            </div>
-
-                            <div>
-                                <div className="mt-1">
-                                    <input
-                                        id="email"
-                                        name="email"
-                                        type="email"
-                                        autoComplete="email"
-                                        defaultValue={invitedEmail}
-                                        placeholder="Email"
-                                        required
-                                        readOnly={!isEnterprise()}
-                                        className={`${isEnterprise() ? '' : 'cursor-not-allowed outline-none border-transparent focus:border-transparent focus:ring-0 border-none '}bg-bg-black text-text-light-gray block h-11 focus:outline-none w-full appearance-none rounded-md px-3 py-2 text-[14px] shadow-sm`}
```

Comment:
How is enterprise handled now?

Reply:
Mmh I didn't noticed that, the only diff is that they can change the email?

Reply:
changed to match the condition, but I would love to know why it's like that